src/main/java/ee/carlrobert/codegpt/credentials/AzureCredentialsManager.java

```diff
@@ -1,70 +1,50 @@
 package ee.carlrobert.codegpt.credentials;
 
 import com.intellij.credentialStore.CredentialAttributes;
-import com.intellij.openapi.application.ApplicationManager;
-import com.intellij.openapi.components.Service;
 import ee.carlrobert.codegpt.settings.state.AzureSettingsState;
+import java.util.Map;
+import java.util.Set;
 import org.jetbrains.annotations.Nullable;
 
-@Service
```

Comment:
Hmm, what's the reason behind removing the service tag? In my opinion, these credential manages should be initialized only once, since they mostly contain logic around the Credentials Store API.